cypress/e2e/write-prescription-spec.cy.js

```diff
@@ -18,32 +18,33 @@ const ourStub = JSON.stringify({
 
 const tvmazeIndividualStub = tvmazeSearchStub
 
-beforeEach(()=>{
-  cy.intercept("https://api.tvmaze.com/singlesearch/shows?q=game+of+thrones", tvmazeSearchStub)
-  cy.intercept("POST", "https://tasterx-api.herokuapp.com/prescriptions/", ourStub)
-  cy.intercept("GET", "https://tasterx-api.herokuapp.com/prescriptions/1", ourStub)
-
-  cy.intercept("https://api.tvmaze.com/shows/82", tvmazeIndividualStub)
-
-})
 
 describe('prescription form', () => {
+  beforeEach(()=>{
+    cy.intercept("https://api.tvmaze.com/singlesearch/shows?q=game+of+thrones", tvmazeSearchStub)
+    cy.intercept("POST", "https://tasterx-api.herokuapp.com/prescriptions/", ourStub)
+    cy.intercept("GET", "https://tasterx-api.herokuapp.com/prescriptions/1", ourStub)
+    cy.intercept("https://api.tvmaze.com/shows/82", tvmazeIndividualStub)
+  })
+
   it('loads the page', () => {
     cy.visit('http://localhost:3000/')
-  })
+  })  
+
   it('lets the user search for a show by name and displays show info when submitted', () => {
-    cy.get("input").type("game of thrones").next().click()
+    cy.get(".search-input").type("game of thrones").next().click()
```

Comment:
looks like you had to be specific on these inputs because there are now 2 that exist on this form.  Good to know!